File: src/wsTripXML/Code/OTAClasses/IssueMCORQ.vb
Option Strict Off
Option Explicit On

Imports System.Xml.Serialization

Namespace wsTravelTalk.wmIssueMCOIn

    '<remarks/>
    <System.Xml.Serialization.XmlRootAttribute(IsNullable:=False)>
    Public Class TT_IssueMCORQ

        '<remarks/>
        Public POS As wmIssueMCOModels.POS

        '<remarks/>
        Public UniqueID As wmIssueMCOModels.UniqueID

        '<remarks/>
        Public ConversationID As String

        '<remarks/>
        Public MCOs() As wmIssueMCOModels.MCO

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute(),
         System.ComponentModel.DefaultValueAttribute(wmIssueMCOModels.Target.Production)>
        Public Target As wmIssueMCOModels.Target = wmIssueMCOModels.Target.Production

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public TransactionStatusCode As wmIssueMCOModels.TransactionStatusCode

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public EchoToken As String

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public TimeStamp As Date

        '<remarks/>
        <System.Xml.Serialization.XmlIgnoreAttribute()>
        Public TimeStampSpecified As Boolean

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public Version As Double

        '<remarks/>
        <System.Xml.Serialization.XmlIgnoreAttribute()>
        Public VersionSpecified As Boolean

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public PrimaryLangID As String

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public AltLangID As String

        '<remarks/>
        <System.Xml.Serialization.XmlAttributeAttribute()>
        Public MaxResponses As Integer

        '<remarks/>
        <System.Xml.Serialization.XmlIgnoreAttribute()>
        Public MaxResponsesSpecified As Boolean


    End Class
End Namespace

File: src/wsTripXML/Code/OTAClasses/IssueMCORQ.cs
using System;
using System.Xml.Serialization;

namespace wsTripXML.wsTravelTalk.wmIssueMCOIn
{

    [XmlRoot(IsNullable = false)]
    public class TT_IssueMCORQ
    {

        public wmIssueMCOModels.POS POS;

        public wmIssueMCOModels.UniqueID UniqueID;

        public string ConversationID;

        public wmIssueMCOModels.MCOMask[] MCOs;

        [XmlAttribute()]
        [System.ComponentModel.DefaultValue(wmIssueMCOModels.Target.Production)]
        public wmIssueMCOModels.Target Target = wmIssueMCOModels.Target.Production;

        [XmlAttribute()]
        public wmIssueMCOModels.TransactionStatusCode TransactionStatusCode;

        [XmlAttribute()]
        public string EchoToken;

        [XmlAttribute()]
        public DateTime TimeStamp;

        [XmlIgnore()]
        public bool TimeStampSpecified;

        [XmlAttribute()]
        public double Version;

        [XmlIgnore()]
        public bool VersionSpecified;

        [XmlAttribute()]
        public string PrimaryLangID;

        [XmlAttribute()]
        public string AltLangID;

        [XmlAttribute()]
        public int MaxResponses;

        [XmlIgnore()]
        public bool MaxResponsesSpecified;


    }
}